cli_li3ds/import_platform.py

```diff
@@ -85,3 +97,29 @@ def take_action(self, parsed_args):
         objs.add(pconfig)
         objs.get_or_create()
         self.log.info('Success!\n')
+
+    @staticmethod
+    def read_lidar_rigid_transform_params(filename):
+        config = configparser.ConfigParser()
+        config.read(filename)
+
+        section = 'TerraMatch corrections v2'
+        easting = config.getfloat(section, 'EastingShift')
+        northing = config.getfloat(section, 'NorthingShift')
+        elevation = config.getfloat(section, 'ElevationShift')
+        heading = config.getfloat(section, 'HeadingShift')
+        roll = config.getfloat(section, 'RollShift')
+        pitch = config.getfloat(section, 'PitchShift')
+
+        t0 = math.cos(heading * 0.5)
+        t1 = math.sin(heading * 0.5)
+        t2 = math.cos(roll * 0.5)
+        t3 = math.sin(roll * 0.5)
+        t4 = math.cos(pitch * 0.5)
+        t5 = math.sin(pitch * 0.5)
+        qw = t0 * t2 * t4 + t1 * t3 * t5
+        qx = t0 * t3 * t4 - t1 * t2 * t5
+        qy = t0 * t2 * t5 + t1 * t3 * t4
+        qz = t1 * t2 * t4 - t0 * t3 * t5
+
```

Comment:
Thanks.

This makes me wonder about Sbet files. Is the same convention used? I need to verify that this query is correct: https://github.com/LI3DS/api-li3ds/blob/master/api_li3ds/apis/foreignpc.py#L436-L467.